torch/quantization/fx/quantize.py

```diff
@@ -496,6 +497,7 @@ def is_observed(input_arg):
 
         model = GraphModule(model, observed_graph)
```

Comment:
following the existing guidance, but seems like directly instantiating `ObservedGraphModule` might also make sense here